easybuild/easyconfigs/p/pyABC/pyABC-0.10.4-foss-2019b-Python-3.7.4.eb

```diff
@@ -0,0 +1,89 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Authors::   Dugan Witherick (University of Warwick)
+# License::   MIT/GPL
+# $Id$
+#
+##
+
+easyblock = 'PythonBundle'
+
+name = 'pyABC'
+version = '0.10.4'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/icb-dcm/pyabc'
+description = """
+Massively parallel, distributed and scalable ABC-SMC (Approximate Bayesian
+Computation - Sequential Monte Carlo) for parameter estimation of complex
+stochastic models.  Implemented in Python with support of the R language.
+"""
+
+toolchain = {'name': 'foss', 'version': '2019b'}
+
+dependencies = [
+    ('Python', '3.7.4'),
+    ('SciPy-bundle', '2019.10', versionsuffix),
+    ('Flask', '1.1.2', versionsuffix),
+    ('bokeh', '1.4.0', versionsuffix),
+    ('dask', '2.8.0', versionsuffix),
+    ('GitPython', '3.1.0', versionsuffix),
+    ('scikit-learn', '0.21.3', versionsuffix),
+    ('IPython', '7.9.0', versionsuffix),
+    ('Arrow', '0.16.0', versionsuffix),
+    ('rpy2', '3.2.6', versionsuffix),
+]
+
+github_account = 'icb-dcm'
+
+exts_default_options = {'source_urls': [PYPI_SOURCE]}
+
+use_pip = True
+
+exts_list = [
+    ('visitor', '0.1.3', {
+        'checksums': ['2c737903b2b6864ebc6167eef7cf3b997126f1aa94bdf590f90f1436d23e480a'],
+    }),
+    ('dominate', '2.4.0', {
+        'checksums': ['a92474b4312bd8b4c1789792f3ec8c571cd8afa8e7502a2b1c64dd48cd67e59c'],
+    }),
+    ('Flask-Bootstrap', '3.3.7.1', {
+        'modulename': 'flask_bootstrap',
+        'checksums': ['cb08ed940183f6343a64e465e83b3a3f13c53e1baabb8d72b5da4545ef123ac8'],
+    }),
+    ('redis', '3.4.1', {
+        'checksums': ['0dcfb335921b88a850d461dc255ff4708294943322bd55de6cfd68972490ca1f'],
+    }),
+    ('dill', '0.3.1.1', {
+        'checksums': ['42d8ef819367516592a825746a18073ced42ca169ab1f5f4044134703e7a049c'],
+    }),
+    ('SQLAlchemy', '1.3.13', {
+        'modulename': 'sqlalchemy',
+        'checksums': ['64a7b71846db6423807e96820993fa12a03b89127d278290ca25c0b11ed7b4fb'],
+    }),
+    ('Pygments', '2.5.2', {
+        'modulename': 'pygments',
+        'checksums': ['98c8aa5a9f778fcd1026a17361ddaf7330d1b7c62ae97c3bb0ae73e0b9b6b0fe'],
+    }),
+    ('feather-format', '0.4.0', {
+        'modulename': 'feather',
+        'checksums': ['a7a0fbc727e1877354d4b4ad83f8e382a249f116241d0ccfc624cac24bddb1a9'],
+    }),
+    (name, version, {
+        'extras': 'R',
+        'patches': ['pyABC-0.10.4-remove_bkcharts_dep.patch'],
```

Comment:
Maybe it's better to filter out the `bkcharts` requirement via `preinstallopts` rather than creating a patch file for it?

Something like this (untested):

```python
# strip out unrequired bkcharts requirement, see https://github.com/ICB-DCM/pyABC/issues/326
'preinstallopts': """sed -i 's/"bkcharts>=0.2",//g' setup.py && """

Reply:
Sure, I'll give it a go and update.